test/Microsoft.Net.Http.Server.FunctionalTests/ResponseSendFileTests.cs

```diff
@@ -34,9 +34,9 @@ public ResponseSendFileTests()
             {
                 var responseTask = SendRequestAsync(address);
 
-                var context = await server.AcceptAsync();
+                var context = await server.AcceptAsync(Utilities.DefaultTimeout);
                 await Assert.ThrowsAsync<FileNotFoundException>(() => 
-                    context.Response.SendFileAsync("Missing.txt", 0, null, CancellationToken.None));
+                    context.Response.SendFileAsync("Missing.txt", 0, null, CancellationToken.None)).TimeoutAfter(Utilities.DefaultTimeout);
```

Comment:
We don't understand why this would every hang. It's only on Win7, correct? Remove this workaround, disable the test on win7, and file a bug.